src/pages/infopages/PrivacyPolicy.tsx

```diff
@@ -1,26 +1,64 @@
-import React, { FC } from 'react';
+import React from 'react';
 import styled from 'styled-components';
-import { useTranslation } from 'react-i18next';
-import Heading from '../../components/Heading';
+import { Trans, useTranslation } from 'react-i18next';
+import { Table, TableBody, TableCell, TableHead, TableRow, Typography } from '@material-ui/core';
 
-import Card from '../../components/Card';
-import { StyledInformationWrapper, StyledNormalTextPreWrapped } from '../../components/styled/Wrappers';
+import { PageHeader } from '../../components/PageHeader';
 
-const StyledHeading = styled(Heading)`
-  text-align: center;
-  padding-bottom: 1rem;
+const StyledPrivacyContainer = styled.div`
+  display: block;
 `;
 
-const PrivacyPolicy: FC = () => {
-  const { t } = useTranslation('infopages');
+const PrivacyPolicy = () => {
+  const { t } = useTranslation('privacy');
 
   return (
-    <StyledInformationWrapper data-testid="privacy-policy" aria-label={t('privacy_policy.heading')}>
-      <Card>
-        <StyledHeading>{t('privacy_policy.heading')}</StyledHeading>
-        <StyledNormalTextPreWrapped>{t('privacy_policy.description')}</StyledNormalTextPreWrapped>
-      </Card>
-    </StyledInformationWrapper>
+    <>
+      <PageHeader>{t('privacy_statement')}</PageHeader>
+
+      <StyledPrivacyContainer>
+        <Typography variant="h2">{t('about.heading')}</Typography>
+        <Typography>{t('about.description')}</Typography>
+
+        <Typography variant="h2">{t('what_are_personal_data.heading')}</Typography>
+        <Typography>
+          <Trans t={t} i18nKey="what_are_personal_data.description">
+            <a target="_blank" rel="noreferrer" href="https://lovdata.no/lov/2018-06-15-38/§2">
+              (i18n content: Personal Data Act Section 2)
+            </a>
+            <a target="_blank" rel="noreferrer" href="https://lovdata.no/lov/2018-06-15-38/gdpr/a5">
+              (i18n content: GDPR Article 5)
+            </a>
```

Comment:
Finnes disse linkene på engelsk også?

Reply:
fra lovdata.no: "The translations are not official; they are provided for information purposes only. In the event of any inconsistency, the Norwegian version shall prevail."

så man kan finne "veiledende" oversettelser på engelsk for noen ting, men norsk er fasit. Hege har linket til norsk side både i norsk og engelsk dokument, så da gjør jeg bare det samme for nå